scripts.v3/utils.js

```diff
@@ -6,6 +6,7 @@ const { BlobServiceClient } = require("@azure/storage-blob");
 const blobStorageContainer = "content";
 const mime = require("mime");
 const apiVersion = "2020-06-01-preview"; // "2021-01-01-preview"; 
+const apiVersion2019 = "2019-12-01";
```

Comment:
```
const apiVersion = "2020-06-01-preview"; // "2021-01-01-preview"; 
const apiVersion2019 = "2019-12-01";`
```

@SSanjeevi, please let's not use different API versions in the script, this pattern may lead to adverse results in the long term. Using `apiVersion` should be enough.

Reply:
done